x-pack/filebeat/module/zoom/webhook/_meta/fields.yml

```diff
@@ -0,0 +1,647 @@
+    - name: sub_account_id
+      type: keyword
+      description: >
+        Related Sub Account
+    - name: operator_id
+      type: keyword
+      description: >
+        UserID that triggered the event
+    - name: operator
+      type: keyword
+      description: >
+        Username/Email related to the user that triggered the event
+    - name: account_id
+      type: keyword
+      description: >
+        Related accountID to the event
+    - name: timestamp
+      type: date
+      description: >
+        Timestamp related to the event
+    - name: time_stamp
+      type: date
+      description: >
+        Timestamp related to the event
+    - name: creation_type
+      type: keyword
+      description: >
+        Creation type
+    - name: account.owner_id
+      type: keyword
+      description: >
+        UserID of the related user the action was performed on
+    - name: account.email
+      type: keyword
+      description: >
+        Email related to the user the action was performed on
+    - name: account.owner_email
+      type: keyword
+      description: >
+        Email related to a sub account which was disassociated
+    - name: account.account_name
+      type: keyword
+      description: >
+        When an account name is updated, this is the new value set
+    - name: account.account_alias
+      type: keyword
+      description: >
+        When an account alias is updated, this is the new value set
+    - name: account.account_support_name
+      type: keyword
+      description: >
+        When an account support_name is updated, this is the new value set
+    - name: account.account_support_email
+      type: keyword
+      description: >
+        When an account support_email is updated, this is the new value set
+    - name: chat_channel.name
+      type: keyword
+      description: >
+        The name of the channel that has been added/modified/deleted
+    - name: chat_channel.id
+      type: keyword
+      description: >
+        The ID of the channel that has been added/modified/deleted
+    - name: chat_channel.type
+      type: keyword
+      description: >
+        Type of channel related to the event. Can be 1(Invite-Only), 2(Private) or 3(Public)
+    - name: chat_channel.timestamp
+      type: date
+      description: >
+        Date and time when channel was created/modified/deleted
+    - name: chat_channel.date_time
+      type: date
+      description: >
+        Epoch time when channel was created/modified/deleted
+    - name: chat_message.id
+      type: keyword
+      description: >
+        Unique ID of the related chat message
+    - name: chat_message.type
+      type: keyword
+      description: >
+        Type of message, can be either "to_contact" or "to_channel"
+    - name: chat_message.date_time
+      type: date
+      description: >
+        Date and time when message was created/modified/deleted
+    - name: chat_message.timestamp
```

Comment:
some of these and the other message type time-related fields seem like good candidates for `@timestamp`

Reply:
The issue will really be to drill down on which, it's very inconsistent even related to similar events, will try to research it a bit more.